redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/YogaUIView.kt

```diff
@@ -7,42 +7,79 @@
 package app.cash.redwood.layout.uiview
 
 import app.cash.redwood.layout.api.Constraint
+import app.cash.redwood.yoga.FlexDirection
 import app.cash.redwood.yoga.MeasureCallback
 import app.cash.redwood.yoga.MeasureMode
 import app.cash.redwood.yoga.Node
 import app.cash.redwood.yoga.Size
 import kotlinx.cinterop.CValue
+import kotlinx.cinterop.ObjCClass
 import kotlinx.cinterop.cValue
 import kotlinx.cinterop.useContents
 import platform.CoreGraphics.CGRectMake
 import platform.CoreGraphics.CGRectZero
 import platform.CoreGraphics.CGSize
 import platform.CoreGraphics.CGSizeMake
+import platform.UIKit.UILabel
+import platform.UIKit.UIScrollView
+import platform.UIKit.UIScrollViewContentInsetAdjustmentBehavior
 import platform.UIKit.UIView
 import platform.UIKit.UIViewNoIntrinsicMetric
 
-internal class YogaUIView : UIView(cValue { CGRectZero }) {
+internal class YogaUIView : UIScrollView(cValue { CGRectZero }) {
   val rootNode = Node()
 
   var width = Constraint.Wrap
   var height = Constraint.Wrap
 
-  var applyModifier: (Node, Int) -> Unit = { _, _ -> }
+  var applyModifier: (Node, Int) -> Unit = { _, _ -> println("applyModifier")}
+
+  init {
+    // TODO: Support Scroll Indicators
+    showsVerticalScrollIndicator = false
```

Comment:
The issue with these atm is that they're part of the view hierarchy. when YogaUIView goes to `syncNodes`, it doesn't know how to differentiate or account for these scroll indicator views